rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -26,10 +36,36 @@
 @pytest.mark.parametrize('accounting_initialize_parameters', [True])
 def test_paraswap_swap_with_fee(accountant: 'Accountant'):
     """Test that the fee in paraswap is handled correctly during accounting"""
-    tx_hash, user_address, contract_address, swap_amount_str, fee_amount_str, receive_amount_str = make_evm_tx_hash(), make_evm_address(), make_evm_address(), '138.2851', '11.308417', '1190.359719'  # noqa: E501
+    tx_hash, user_address, contract_address, aquired_from_address, swap_amount_str, fee_amount_str, receive_amount_str = make_evm_tx_hash(), make_evm_address(), make_evm_address(), make_evm_address(), '138.2851', '11.308417', '1190.359719'  # noqa: E501
     events = [EvmEvent(
+        tx_hash=tx_hash,
+        sequence_index=0,
+        timestamp=TIMESTAMP_0_MS,  # half price at t=0
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.RECEIVE,
+        event_subtype=HistoryEventSubType.NONE,
+        asset=A_WBTC,
+        balance=Balance(amount=FVal(swap_amount_str)),
+        location_label=user_address,
+        notes=f'Receive {swap_amount_str} WBTC from {aquired_from_address} to {user_address}',
+        counterparty=CPT_PARASWAP,
```

Comment:
Hey can you make this into another random address?

Makes the reader think it's actually paraswap related while it's not. Also add a comment this event is to create cost basis for WBTC

```suggestion
        counterparty=0xrandomaddress
```